tests/test_autodiscover.py

```diff
@@ -50,8 +50,6 @@ class TestImportLibraries:
         }
     )
     def test_import_libraries(self):
-        # Ensure we start with a clean state
-        registry.clear()
```

Comment:
For unknown reason this was throwing errors when running both this test file and `tests/test_component.py`. But luckily it's fine to remove it.